src/platform/ESP32/PlatformManagerImpl.cpp

```diff
@@ -97,6 +99,7 @@ CHIP_ERROR PlatformManagerImpl::_InitChipStack(void)
     esp_wifi_get_mode(&mode);
```

Comment:
Shouldn't this be under the ifdef as well? Also, `wifi_mode_t mode;` above

Reply:
Done